key.js

```diff
@@ -1,277 +1,446 @@
-// Button Animation Logic :
 
-const buttons = document.querySelectorAll('.btns');
-const dUp = document.querySelector('#up')
-const dLeft = document.querySelector('#left')
-const dRight = document.querySelector("#right")
-const dDown = document.querySelector("#down")
-const headings = document.querySelector('h1');
-const timerElement = document.querySelector(`h6`);
-const keyBoard = document.querySelector(`.contents`);
-const startButton = document.querySelector('.main');
-let seconds = 0;
-const levelArray = [`LEVEL 1 >>`, `LEVEL 2 >> `, `LEVEL 3 >> `, `LEVEL 4 >> `, `LEVEL 5 >>`];
-let lives = 3; // 3 LIVES by Default
+// GAME STATE INFO :
+// 0. Assign this IIFE's return value to a variable
 
-const buttonPress = () => {
-    const audio = new Audio("GameAudio/buttonPress.wav");
-    audio.currentTime = 0;
-    audio.play();
-}
 
-const setComplete = () => {
-    const audio = new Audio("GameAudio/setComplete.wav");
-    audio.currentTime = 0;
-    audio.play();
-}
-const invalidInput = () => {
-    const audio = new Audio('GameAudio/invalid.flac')
-    audio.currentTime = 0;
-    audio.play();
-}
-const clickSoundGreen = () => {
-    const audio = new Audio('GameAudio/greenSound2.mp3');
-    audio.currentTime = 0;
-    audio.play();
-}
-const clickSoundRed = () => {
-    const audio = new Audio('GameAudio/MPOP.wav')
-    audio.currentTime = 0;
-    audio.play();
-}
-// KeyPress Event :
+const gameModule = (function () {
+    const getInitialGameState = () => ({
+        lives: 3,
+        seconds: 60,
+        levelsInfo: {
+            currentLevel: 0,
+            totalLevels: 5,
+            setInfoList: [1, 2, 3, 6, 8, 11],
+            currentSetValue: 1
+        },
+        isStart: false,
+        isGameOver: false,
+        sound: true,
+        gameAudio: {
+            getGameAudio: new Audio("GameAudio/gameBackggameState.gameAudio.getStartSound.play();roundMusic.mp3"),
+            getClickAudio: new Audio("GameAudio/buttonPress.wav"),
+            getStartSound: new Audio("GameAudio/start_sound.wav"),
+            getSetCompleteSound: new Audio("GameAudio/setComplete.wav"),
+            getKeyboardSounds: {
+                getInvalidAudio: new Audio('GameAudio/invalid.flac'),
+                getGreenAudio: new Audio()(`GameAudio/greenSound2.mp3`),
+                getRedAudio: new Audio("GameAudio/MPOP.wav")
+            }
```

Comment:
Remove `get` from the prefix of all the keys and rename them:

```suggestion
            gameAudio: new Audio("GameAudio/gameBackggameState.gameAudio.getStartSound.play();roundMusic.mp3"),
            clickAudio: new Audio("GameAudio/buttonPress.wav"),
            startSound: new Audio("GameAudio/start_sound.wav"),
            setCompleteSound: new Audio("GameAudio/setComplete.wav"),
            keyboardSounds: {
                invalidAudio: new Audio('GameAudio/invalid.flac'),
                greenAudio: new Audio()(`GameAudio/greenSound2.mp3`),
                redAudio: new Audio("GameAudio/MPOP.wav")
            }
```